README.md

```diff
@@ -19,22 +18,32 @@ A comprehensive, high performance, easy to use, open source animation library in
 * **Good synchronisation** between tweens
 * **Relative tweening** enables the possibility to alter objects while they are tweening
 * **Nested object tweening** enables the possibility to alter nested objects using a single tween
+* **Compatible with Node.js!**
 * Bonus: Creation and removal of tweens within the callback of another tween will not result in any unwanted side effect (infamous bug of other tweening libraries)
 
 Note: Do not hesitate to contribute by reporting issues or by submitting your own components and interpolations.
 
+Warning: Still in beta version!
+
 ## How to use
 
-Include tina's build in your html using either the [minified library](https://raw.githubusercontent.com/Wizcorp/tina/master/build/tina.min.js) or the [unminified version](https://raw.githubusercontent.com/Wizcorp/tina/master/build/tina.js).
+### In a browser
+Include TINA's build in your html using either the [minified library](https://raw.githubusercontent.com/Wizcorp/tina/master/build/tina.min.js) or the [unminified version](https://raw.githubusercontent.com/Wizcorp/tina/master/build/tina.js).
 
 ```html
 <script src="tina.min.js"></script>
 ```
 
+### In Node.js
+Install TINA using ```npm install tina```, then require it:
+```javascript
+var TINA = require('tina');
+```
+
 ## API
 
-Existing playable components are: Tween, NestedTween, Timeline, Sequence, Delay, Timer, Ticker (Recorder coming soon).
-The following is a non-exhaustive list of possibilities offered by the API.
+Existing playable components are: **Tween, NestedTween, Timeline, Sequence, Delay, Timer, Ticker** (Recorder and CSS Tween coming soon).
+The following is a non-exhaustive list of possibilities offered by the TINA.
```

Comment:
lol, it used to say "the API", where "the" made sense.. but hey, you go for it ;)